app/views/tags/edit.html.erb

```diff
@@ -9,6 +9,10 @@
 
 <!--main content-->
 
+<% if logged_in_as_admin? %>
+  <p><%= ts("Last updated by") %> <%= @tag.last_wrangler.try(:login) || '---' %> <%= ts("on") %> <%= @tag.updated_at %></p>
```

Comment:
If you have a minute, could you update the way this is coded?

`<%= ts("Last updated by %{wrangler} on %{date}", wrangler: @tag.last_wrangler.try(:login) || '---', date: @tag.updated_at) %>`